state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -626,14 +643,38 @@ pub mod pallet {
 		DepositBoosted {
 			deposit_address: TargetChainAccount<T, I>,
 			asset: TargetChainAsset<T, I>,
-			amount: TargetChainAmount<T, I>,
+			amounts: BTreeMap<BoostPoolTier, TargetChainAmount<T, I>>,
 			deposit_details: <T::TargetChain as Chain>::DepositDetails,
+			boost_id: BoostId,
+			channel_id: ChannelId,
 			// Ingress fee in the deposit asset. i.e. *NOT* the gas asset, if the deposit asset is
-			// a non-gas asset.
+			// a non-gas asset. The ingress fee is taken *after* the boost fee.
 			ingress_fee: TargetChainAmount<T, I>,
+			// Total fee the user paid for their deposit to be boosted.
 			boost_fee: TargetChainAmount<T, I>,
 			action: DepositAction<T::AccountId>,
 		},
+		BoostFundsAdded {
+			account_id: T::AccountId,
```

Comment:
We sometimes use booster_id and sometimes account_id. Why not use booster_id everywhere we talk about boosting?